### PR TITLE
chore(pie-monorepo): ci/cd renovate grouping 

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Changed
- Github workflow to allow changesets to generate via renovate bot.

### Added
- `renovate.json` grouping config.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      "enabled": true
+    },
+    {
+      "description": "Group dependencies from package.json within apps directory",
+      "matchFiles": ["apps/pie-microsite/package.json"],
+      "matchDepTypes": ["devDependencies", "dependencies"],
+      "matchPackagePatterns": ["markdown-*"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "markdown",
+      "groupSlug": "root-markdown-devDeps",
```
Comment (siggerzz) [inline renovate.json:78]: Probably worth changing this to `deps` as it includes both types

File: renovate.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      "enabled": true
+    },
+    {
+      "description": "Group dependencies from package.json within apps directory",
+      "matchFiles": ["apps/pie-storybook/package.json"],
+      "matchDepTypes": ["devDependencies", "dependencies"],
+      "matchPackagePatterns": ["@storybook/*", "storybook"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "storybook",
+      "groupSlug": "root-storybook-devDeps",
```
Comment (siggerzz) [inline renovate.json:88]: ditto

**Review by siggerzz — CHANGES_REQUESTED** [review]
Minor comments!

File: renovate.json (hunk 1 above)
Comment (kevinrodrigues) [reply renovate.json]: Fixed.

File: renovate.json (hunk 2 above)
Comment (kevinrodrigues) [reply renovate.json]: Fixed.

**Comment by kevinrodrigues** [comment]
> Minor comments!

@siggerzz all done, cheers!

**Review by JoshuaNg2332 — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Pull request merged**